fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/catalog/FlinkCatalogITCase.java

```diff
@@ -512,31 +512,6 @@ void testCreateDatabase() {
         assertThat(databases.toString()).isEqualTo(String.format("[+I[%s]]", DEFAULT_DB));
     }
 
-    @Test
-    void testFactoryCannotFindForCreateTemporaryTable() {
```

Comment:
The tests are added on purpose to force accessing Fluss tables via catalog. So we didn't add `com.alibaba.fluss.flink.catalog.FlinkTableFactory` entry to SPI.